msal/src/Microsoft.Identity.Client/AccountId.cs

```diff
@@ -80,6 +80,14 @@ public AccountId(string identifier, string objectId, string tenantId)
             ValidateId();
         }
 
+        /// <summary>
+        /// Constructor of an AccountId meant for Adfs scenarios since Adfs instances lack tennt id's.
```

Comment:
>tennt id's [](start = 91, length = 10)

typos (tennt / ids) Also CC @jmprieur as he reviews all public docs. I wonder if the input param of this ctor should be adfsIdentifier to make intent more clear...